java/build.gradle

```diff
@@ -39,15 +38,9 @@ dependencies {
     annotationProcessor("com.google.auto.service:auto-service:1.0")
     compileOnly("com.google.auto.service:auto-service:1.0")
 
-    // custom jaeger-thrift based exporter
-    implementation("io.opentelemetry:opentelemetry-exporter-jaeger-thrift")
-    implementation("io.jaegertracing:jaeger-thrift:1.5.0")
```

Comment:
We're dropping jaeger 😮 
Does it mean that only otlp/http is supported right now? Is that in line with the GDI spec?

Reply:
According to Steve since Splk ingest understands OTLP the only reason for J/T was comp with SFx SmartAgent. This will never be the case in lambda env (direct ingest now, in the future - OTel Collector as well). Therefore it does not make sense to keep J/T support, especially taking into account lambda size constraints.

Reply:
I'm okay with that as long as GDI spec reflects this change